test/logging.test.js

```diff
@@ -132,12 +132,19 @@ test('format log file log message with level info', () => {
 })
 
 test('displays error log well formatted', () => {
-  displayErrorLog(exampleErrorLog)
+  const extendedExampleErrorLog = [...exampleErrorLog, {
+    ts: new Date('2022-01-01T01:05:43+01:00').toJSON(),
+    level: 'warning',
+    warning: 'another warning'
+  }]
+
+  displayErrorLog(extendedExampleErrorLog)
 
-  expect(consoleLogSpy.mock.calls).toHaveLength(3)
-  expect(consoleLogSpy.mock.calls[0][0]).toContain('The following 1 errors and 1 warnings occurred:')
+  expect(consoleLogSpy.mock.calls).toHaveLength(4)
+  expect(consoleLogSpy.mock.calls[0][0]).toContain('The following 1 errors and 2 warnings occurred:')
```

Comment:
I have adjusted this test to make sure the number of warnings don't match the number of errors. This will prevent similar issues from happening again in the future.